colab/colab_utils.py

```diff
@@ -0,0 +1,1048 @@
+#@title Please run this cell first!
+
+import base64
+import collections
+import datetime
+import io
+import json
+import os
+import re
+import subprocess
+import time
+
+import ee
+import folium
+from folium.plugins import HeatMap
+import ipyplot
+import IPython.display
+import pandas as pd
+import pexpect
+import pprint
```

Comment:
Unused imports (that I found): pprint, pytz, smtplib, ssl, google.appengine.api.mail

These should be removed.

Reply:
Thank you, those are indeed unused and will be removed.